### PR TITLE
86 multihazard issues 22jan

### PR DESCRIPTION
**Pull request by Ssravani272**
- Add the 'Ex' text in description.
- Update the success text in thank you card
- Add link in the success text in thank you card

### CODE REVIEW

File: deployments/ph/assets/locales/en.json [hunk 1]
```diff
@@ -255,8 +255,8 @@
       "urlText":"Mapakalamidad.ph",
       "title": {
         "real" : [
-        "Thank you for your report. Check",
-        "to view your report and see real-time disaster updates."
+        "Thank you for your report.",
+        "We will notify you when it’s on the map."
```
Comment (ruthvik129) [inline deployments/ph/assets/locales/en.json:259]: @Ssravani272 Why did you change it to We will notify you when its on map , also did you update the tagalog version also

Comment (Ssravani272) [reply deployments/ph/assets/locales/en.json:259]: This text was given in updated translations wireframe, so I changed it @ruthvik129

**Review by ruthvik129 — APPROVED** [review]
